lib/common.js

```diff
@@ -13,7 +13,7 @@ const apiKey = process.env.AIRTABLE_API_KEY;
 const baseId = process.env.AIRTABLE_BASE_ID;
 
 const BASE = new Airtable({
-  apiKey
+  apiKey,
```

Comment:
No worries about the trailing comma diffs - it gets updated in #53 (but odd that your prettier is using a different config; I added a workspace `settings.json` in the same PR so hopefully that fixes it)

Reply:
should I leave them as is then?